broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -167,7 +167,6 @@ public static BrokerController buildBrokerController(String[] args) throws Excep
             System.exit(-4);
         }
 
-        messageStoreConfig.setHaListenPort(nettyServerConfig.getListenPort() + 1);
```

Comment:
If the user has not configured it, could the default port for HA be set to nettyServerConfig.getListenPort() + 1 ?

Reply:
haListenPort default value is nettyServerConfig.getListenPort()'default value +1

![image](https://user-images.githubusercontent.com/63851981/230263729-f9e4a30d-99b5-441a-af42-0464d4464fed.png)

Reply:
nettyServerConfig.listenPort default value is 10911

![image](https://user-images.githubusercontent.com/63851981/230264596-867b5f78-86da-452b-8f0d-a326f9cb520b.png)

Reply:
IMO this PR will break the compatibility, that is, the default value of `haLitenPort` is `listenPort + 1`. Users setting `listenPort` but not setting `haLitenPort` explicitly will find `haLitenPort` changes after updating to new version.

The more appropriate way may be: 

- if `haLitenPort` is set explicitly, use the value set by user; 
- if `haLitenPort` is not set, assign `listenPort + 1` to it.